ctapipe_io_lst/tests/test_event_time.py

```diff
@@ -11,13 +11,58 @@
 from ctapipe_io_lst.constants import N_MODULES
 from ctapipe_io_lst.containers import LSTArrayEventContainer
 
-from traitlets.config import Config
-
-
 test_data = Path(os.getenv('LSTCHAIN_TEST_DATA', 'test_data'))
 test_run_summary = test_data / 'real/monitoring/RunSummary/RunSummary_20200218.ecsv'
 
 
+def test_combine_counters():
+    from ctapipe_io_lst.event_time import combine_counters
+
+    pps_counter = np.uint16(60_000)
```

Comment:
just curiosity... is this name of writing integers coming from some pep recommendation?

Reply:
I am not sure I understand. Do you mean the underscore? That's just for readability, it was introduced in 3.6.

Reply:
yes, I meant the underscore, thanks!